docs/src/Groups/permgroup.md

```diff
@@ -30,12 +31,38 @@ projective_unitary_group
 projective_special_unitary_group
 ```
 
+## Operations for permutation groups
+
+All operations, properties and attributes for general groups
+describe in the previous sections are supported for permutation
+groups. In addition there are some specific to permutation group.
+
 In OSCAR, every permutation group has a degree `n`, that corresponds to the size of the set on which `G` acts.
 
 ```@docs
 degree(x::PermGroup)
 ```
 
+```@docs
```

Comment:
The reason there are three `@docs` blocks here is mundane... I copied them here from elsewhere as the changes in this PR evolved. I also thought about merging them, but then the separation seemed to make some sense. Perhaps some text could be added between them to make this clear, e.g.:
````suggestion

The following functions deal with the natural action of a given permutation group $G$.
```@docs
````

Reply:
To be clear, I am neutral as to whether we add these (or other) filler texts; or fuse the `@docs` blocks; or leave them as they are in this PR. If anyone has a preference either way, please say so.